.github/workflows/buildsCI.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
 
 
   #--------------------------------------------------------------------------------
-  Win_SH_EX_CfU:  # Windows, shared, with Examples & Tools, and Check for Updates
+  Win_SH_EX_CfU:  # Windows, shared, with Examples & Tools, no EasyLogging and Check for Updates
```

Comment:
Don't we also need to check on Linux?

Reply:
Since it's at the application level I thought it's enough to test on 1 OS, you think Linux is needed too?
We can add..